packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -240,8 +257,30 @@ export class LitDevSearch extends LitElement {
 
   private _listboxClicked = false;
 
-  update(changed: PropertyValues<this>) {
-    if (!this._searchText && this._inputEl) {
+  private _lastSuggestions: Suggestion[] = [];
+
+  private get _currentSuggestions() {
+    if (!this._searchTask || this._searchTask.status !== TaskStatus.COMPLETE) {
+      return this._lastSuggestions;
+    }
+
+    this._lastSuggestions = this._searchTask.value!;
+    return this._searchTask.value!;
+  }
+
+  private get _isExpanded() {
+    return (
+      this._isFocused &&
+      this._currentSuggestions.length > 0 &&
+      !this._closedByEscape
+    );
+  }
+
+  update(changed: Map<string, unknown>) {
```

Comment:
Minor comment since we don't need `changed` other than to pass into `super.update()`, but was the type change here intentional?

Reply:
So midway I was experimenting with accessing a private property that was changed, and (known issue) `PropertyValues` can't see private properties due to TS limitations. I've since reverted that, but didn't change this back. Do you think it's worth changing back?

Reply:
Eh, I say not a big deal.

Reply:
`PropertyValues` without a generic should be the same